boa/src/class.rs

```diff
@@ -116,7 +116,11 @@ impl<'context> ClassBuilder<'context> {
     where
         T: ClassConstructor,
     {
-        let mut builder = ConstructorBuilder::new(context, T::raw_constructor);
+        let mut builder = ConstructorBuilder::with_standard_object(
```

Comment:
We should not use `with_standard_object` since it will override the standard object in this case `Object` object.

Reply:
Ah, I missed that. I have to create a new object with `[[Prototype]]: Object` and set that new object as the prototype of the builder, correct?

Reply:
> Ah, I missed that. I have to create a new object with `[[Prototype]]: Object` and set that new object as the prototype of the builder, correct?

`ConstructorBuilder` creates a new constructor like `Object()`, `String()`, etc, it does not create a new object instance.